src/database/queries/getData.js

```diff
@@ -27,6 +27,12 @@ const getSpecalize = (categoryId) => {
   return connect.query(sql);
 };
 
+const getUser = (userId) => {
+  const sql = 'SELECT users.specalization_id, users.firstname, users.lastname, users.email, users.freelancer_url, users.photo_url, users.mobile_number, users.birthday, users.city, users.password, specialization.name as specialization, field.name as category, specialization.field_id  from (users join specialization  on users.specalization_id = specialization.id) join field on field.id =specialization.field_id where users.id = $1';
```

Comment:
@Angham116 Since this is being returned to the browser in the settings GET, I think you should remove the password from this select statement just to be safe.

Reply:
@Angham116 thanks for doing that!